README.md

```diff
@@ -188,8 +244,8 @@ cd build-output
 ctest --output-on-failure
 ```
 
-You will find compiled binaries in `build-output\` respective to their source directories.
-
+You will find compiled binaries in `build-output\` respective to their
```

Comment:
Should this be `build-output/` (with a forward slash)?

Reply:
That is the section for Windows, where it is customary to use `\` for a directory separator, right?

Reply:
Bah, GitHub had collapsed the lines between 202 and here -- I thought this was the macOS section.  Nevermind, then :)